server/start.go

```diff
@@ -148,6 +149,8 @@ which accepts a path for the resulting pprof file.
 	cmd.Flags().Uint64(server.FlagMinRetainBlocks, 0, "Minimum block height offset during ABCI commit to prune Tendermint blocks")
 	cmd.Flags().String(srvflags.AppDBBackend, "", "The type of database for application and snapshots databases")
 
+	cmd.Flags().String(srvflags.QueryOnly, "", "Start the node in query only mode without Tendermint process (grpc|jsonrpc|grpc,jsonrpc)")
```

Comment:
Feel like it might make more sense to make this flag `QueryOnly` a `bool` instead of a string, assigned to `true` if the node is supposed to be started in query-only mode w/ Tendermint (whether gRPC or JSON-RPC) and `false` otherwise.

Reply:
seems you're reviewing before this [updated](https://github.com/evmos/ethermint/pull/1360/commits/d1bb97e80ec22d2a1f3cac0a7bcf69ba2c28ce26)